inspire_query_parser/visitors/elastic_search_visitor.py

```diff
@@ -448,6 +448,17 @@ def _generate_query_string_query(self, value, fieldnames, analyze_wildcard):
 
         return query
 
+    def _generate_wildcard_query(self, value, fieldnames):
+        if not fieldnames:
+            fieldnames = ['_all']
+        if not isinstance(fieldnames, list):
+            fieldnames = [fieldnames]
+        should_query = []
+        for field in fieldnames:
+            query = {"wildcard": {field: {"value": value}}}
+            should_query.append(query)
```

Comment:
FYI this kind of pattern is a list comprehension:
```suggestion
        should_query = [{"wildcard": {field: {"value": value}}} for field in fieldnames]
```
I like list comprehensions, some people don't, do as you like.